js/sq2display.js

```diff
@@ -845,6 +826,13 @@ function displaysq2MeleeDamage(parent_elem, overallparent_elem, meleeStats){
     critStats.append(critChance);
 
     parent_elem.append(critStats);
+
+    //up and down arrow - done ugly
+    let arrow = document.createElement("img");
+    arrow.id = "arrow_" + overallparent_elem.id;
+    arrow.style.maxWidth = document.body.clientWidth > 900 ? "3rem" : "10rem";
+    arrow.src = "../media/icons/" + (newIcons ? "new" : "old") + "/toggle_down.png";
+    overallparent_elem.appendChild(arrow);
```

Comment:
Can this code be combined with the existing arrow code to reduce duplication? ig just separate function prolly so its standardized if we ever decide to change them

Reply:
Yea should've done that oops

Reply:
res?